src/Umbraco.Core/Persistence/Migrations/Upgrades/TargetVersionSevenFourZero/EnsureContentTypeUniqueIdsAreConsistent.cs

```diff
@@ -0,0 +1,42 @@
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Persistence.SqlSyntax;
+
+namespace Umbraco.Core.Persistence.Migrations.Upgrades.TargetVersionSevenFourZero
+{
+    /// <summary>
+    /// Courier on v. 7.4+ will handle ContentTypes using GUIDs instead of
+    /// alias, so we need to ensure that these are initially consistent on
+    /// all environments (based on the alias).
+    /// </summary>
+    [Migration("7.4.0", 2, GlobalSettings.UmbracoMigrationName)]
+    public class EnsureContentTypeUniqueIdsAreConsistent : MigrationBase
+    {
+        public EnsureContentTypeUniqueIdsAreConsistent(ISqlSyntaxProvider sqlSyntax, ILogger logger)
+            : base(sqlSyntax, logger)
+        {
+        }
+        
+        public override void Up()
+        {
+            var sql = @"select umbracoNode.id,
```

Comment:
@clausjensen It's much better to build this up with our Sql object which takes care of SQL parameters (i.e. no sql injection) and takes care of quoting column names correctly between SQL server types. I'll fix up and show you.